.github/workflows/conan-build.yml

```diff
@@ -289,30 +264,10 @@ jobs:
     name: GCC ${{ matrix.gcc-version }} - ${{ matrix.os-name }} - ${{ matrix.build-type }}
     runs-on: ${{ matrix.os }}
     container:
-      image: ubuntu:25.04
+      image: linevelynn/gcc-15-conan-aui:latest
```

Comment:
shall we move this under github.com/aui-framework namespace?

Reply:
I have created a repo for this.

https://github.com/aui-framework/docker-images

Please upload your Dockerfiles there and setup a pipeline to build & push them to AUI's gh.io (I assume you have such permission, if not, please dm me)

Pipeline example: https://github.com/qemus/qemu/blob/master/.github/workflows/build.yml